src/download.jl

```diff
@@ -7,9 +7,12 @@ Returns the URL of the gpkg dataset of the country
 function dataurl(dataid)
     # Only accepts GADM Dataset Provider
     if contains(dataid, "GADM")
-        provider, country_code = split(dataid, '_')
-
-        "https://biogeo.ucdavis.edu/data/gadm3.6/gpkg/gadm36_$(country_code)_gpkg.zip"
+        try
+            provider, country_code = split(dataid, '_')
+            "https://biogeo.ucdavis.edu/data/gadm3.6/gpkg/gadm36_$(country_code)_gpkg.zip"
+        catch err
+            @error "Invalid dataset format, please use GADM_<country_code>."
+        end
```

Comment:
So this part can be rewritten with a Julia regex. The try-catch shouldn't be needed.